playbooks/byo/openshift-glusterfs/config.yml

```diff
@@ -0,0 +1,6 @@
+---
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
Oh, so that's not part of `std_include.yml`? I imagine that's just another include line below the std-include one?